app/themes/3column/ui/app/root/root.html

```diff
@@ -0,0 +1,43 @@
+<div>
+  <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <ml-user></ml-user>
+      <a class="navbar-brand" href="/"></a>
+    </div>
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <div class="navbar-form navbar-right" ng-show="user.authenticated">
+        <ml-remote-input></ml-remote-input>
+      </div>
+    </div><!-- /.navbar-collapse -->
+  </nav>
+  <!-- Add your site or application content here -->
+  <div class="container-fluid" ui-view></div>
+
+  <footer id="footer" class="powered">
+    <div class="container">
+      <div class="row">
```

Comment:
I'm missing the row with social links here. Copy paste mistake? https://github.com/marklogic/slush-marklogic-node/blob/master/app/templates/ui/app/root/root.html#L32